PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -383,8 +408,17 @@ struct HfEventSelectionMc {
         centralityMin = option.defaultValue.get<float>();
       } else if (option.name.compare("hfEvSel.centralityMax") == 0) {
         centralityMax = option.defaultValue.get<float>();
+      } else if (option.name.compare("hfEvSel.requireGoodRct") == 0) {
+        requireGoodRct = option.defaultValue.get<bool>();
+      } else if (option.name.compare("hfEvSel.rctLabel") == 0) {
+        rctLabel = option.defaultValue.get<std::string>();
+      } else if (option.name.compare("hfEvSel.rctCheckZDC") == 0) {
+        rctCheckZDC = option.defaultValue.get<bool>();
+      } else if (option.name.compare("hfEvSel.rctTreatLimitedAcceptanceAsBad") == 0) {
+        rctTreatLimitedAcceptanceAsBad = option.defaultValue.get<bool>();
       }
     }
+    rctChecker.init(rctLabel, rctCheckZDC, rctTreatLimitedAcceptanceAsBad);
```

Comment:
Also here, we could have a common `init` functions that includes `addHistograms` and `configureFromDevice`.

Reply:
Done